### ISSUE TITLE
Guilded integration

### ISSUE THREAD

**Issue by Haelikor** [Enitoni/pulseshitter#13]
As shitty Roblox knockoff to Discord that Discord "borrows" features from sometime, was wondering how difficult it'd be to do this but with Guilded. It has a bot api now and a similar problem.

**Comment by Emik03** [comment]
This feels like huge scope creep to me, honestly.
Correct me if I am wrong but there doesn't appear to be a client library for sending audio to a bot in Guilded, which would need to be done first before such an application can be made.

**Comment by Enitoni** [comment]
While I am not opposed to the idea of making pulseshitter work with other platforms, there is some substantial work that need to be made to the codebase in order to support this. I would need a very good reason to do it, and currently I don't feel as if this is a good enough reason.

I also don't use Guilded which makes it less important to me. However, if enough people ask for it, I will consider it.

Not to mention, for any platform supported, there must be good, mature, and usable client libraries available. Looking a bit at the crates for guilded, I only found a few related to the same project and they have several issues:
1. They seem to no longer be maintained.
2. They are unfinished and do not have any working examples.
3. There is no voice library, which is of upmost importance when implementing anything related to voice chat.

If these issues are resolved and there is enough of a demand for this sort of support, I will definitely consider it. However, for know I will close this as it seems to be way too out of scope and requires a lot of work for very little perceived gain.
